prov/verbs/src/verbs_msg_ep.c

```diff
@@ -290,14 +291,23 @@ int fi_ibv_open_ep(struct fid_domain *domain, struct fi_info *info,
 
 	fi = dom->info;
 
+	ret = ibv_query_device(dom->verbs, &device_attr);
+	if (ret) {
+		VERBS_INFO(FI_LOG_DOMAIN,
+			   "ibv_query_device failed with errno\n",
+			   errno);
+		return -errno;
+	}
+
```

Comment:
Where is device_attr getting used in the function?

Reply:
Sorry, I forgot to remove this